packages/thirdweb/src/extensions/farcaster/signers.ts

```diff
@@ -0,0 +1,17 @@
+import { fromBytes } from "../../utils/encoding/from-bytes.js";
+import type { Hex } from "../../utils/encoding/hex.js";
+
+export type Ed25519Keypair = {
+  publicKey: Hex;
+  privateKey: Hex;
+};
+
+export async function createEd25519Keypair(): Promise<Ed25519Keypair> {
```

Comment:
for internal functions, mark them as @internal so the linter doesnt yell at you

Reply:
This one is public-facing, it's used to generate new account signers (has to be run on the user's end so they can store the keypair). I renamed it to createSigner for clarity